examples/batch/v1beta1/jobqueue.yaml

```diff
@@ -0,0 +1,335 @@
+apiVersion: batch.aws.upbound.io/v1beta1
+kind: JobQueue
+metadata:
+  annotations:
+    meta.upbound.io/example-id: batch/v1beta1/jobqueue
+  labels:
+    testing.upbound.io/example-name: test_queue
+  name: test-queue
+spec:
+  forProvider:
+    computeEnvironmentOrder:
+      - computeEnvironmentSelector:
+          matchLabels:
+            testing.upbound.io/example-name: sample
+        order: 1
+    priority: 1
+    region: eu-west-1
+    state: ENABLED
+  providerConfigRef:
+    name: aws-provider-config-test     
+
+---
+apiVersion: batch.aws.upbound.io/v1beta1
+kind: ComputeEnvironment
+metadata:
+  annotations:
+    meta.upbound.io/example-id: batch/v1beta1/jobqueue
+  labels:
+    testing.upbound.io/example-name: sample
+  name: sample
+spec:
+  forProvider:
+    computeResources:
+      instanceRoleSelector:
+        matchLabels:
+          testing.upbound.io/example-name: ecs_instance_role
+      instanceType:
+        - c4.large
+      maxVcpus: 16
+      minVcpus: 0
+      placementGroupSelector:
+        matchLabels:
+          testing.upbound.io/example-name: sample_pg
+      securityGroupIdsRefs:
+      - name: sample
+      subnetsRefs:
+      - name: sample
+      type: EC2
+    region: eu-west-1
+    serviceRoleSelector:
+      matchLabels:
+        testing.upbound.io/example-name: aws_batch_service_role
+    type: MANAGED
+  providerConfigRef:
+    name: aws-provider-config-test    
```

Comment:
We use the default provider config in uptest; could you please remove the above?